recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -30,13 +33,16 @@ def _minimum_compilers_version(self):
             "Visual Studio": "16",
         }
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def config_options(self):
+        if tools.Version(self.version) <= "1.1.8":
+            del self.options.disable_module
 
     def configure(self):
         if self.settings.compiler.get_safe("cppstd"):
             tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        if tools.Version(self.version) <= "1.1.8" and self.settings.compiler in ["msvc", "Visual Studio"]:
```

Comment:
It should be moved to validate()

Reply:
Yes, of course. I've moved the checks from the `configure` method to the `validate` method.